<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddAttribute</class>
 <widget class="QDialog" name="CAddAttribute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Custom Build</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Excludes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="dir"/>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" rowspan="3" colspan="2">
    <widget class="QListWidget" name="excludes"/>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Directory:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="delExcludeBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/del.png</normaloff>:/resources/del.png</iconset>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QToolButton" name="delExtraBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/del.png</normaloff>:/resources/del.png</iconset>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QToolButton" name="addExtraBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/plus.png</normaloff>:/resources/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="addExcludeBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/plus.png</normaloff>:/resources/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Extra RSync Options:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" rowspan="3" colspan="2">
    <widget class="QListWidget" name="extras"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dir</tabstop>
  <tabstop>excludes</tabstop>
  <tabstop>addExcludeBtn</tabstop>
  <tabstop>delExcludeBtn</tabstop>
  <tabstop>extras</tabstop>
  <tabstop>addExtraBtn</tabstop>
  <tabstop>delExtraBtn</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CAddAttribute</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CAddAttribute</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
